Makefile

```diff
@@ -35,6 +35,9 @@ imgs_server_dest := $(imgs_source:%=build/server/%)
 
 raven_source := $(shell node -e 'console.log(require.resolve("raven-js/dist/raven.js"))')
 
+l10n_source := $(wildcard locales/*)
+l10n_dest := $(l10n_source:%/webextension.properties=addon/webextension/_locales/%/messages.json)
```

Comment:
@ianb meh, decided to just solve today's problem, and worry about building server strings when we get there